cernopendata/views.py

```diff
@@ -44,6 +44,15 @@ def redefine_search_endpoint(blueprint_setup):
 
 def search_wrapper():
     """Wrap default invenio search endpoint."""
+    # translate old search query params to new format
+    # e.g. type=Dataset => f=type:Dataset
+    facets = current_app.config['RECORDS_REST_FACETS']
+    facet_keys = facets['_all']['aggs'].keys()
```

Comment:
I think storing `facet_keys` in memory on app load could speed things up and save us from parsing  `RECORDS_REST_FACETS` to get the keys on each request. What do you think?

Reply:
Where do you suggest storing them?

Reply:
This can be ignored, since the facets tree is also needed in `translate_search_url`